audit.tf

```
@@ -29,7 +29,9 @@ resource "aws_cloudwatch_event_rule" "monitor_iam_access_audit" {
     userIdentity = jsonencode(each.value.userIdentity)
   })
 
-  depends_on = [data.aws_iam_role.monitor_iam_access_audit, data.aws_iam_user.monitor_iam_access_audit]
+  depends_on = [
+    data.aws_iam_role.monitor_iam_access_audit, data.aws_iam_user.monitor_iam_access_audit
```

These should be split IMO if you're going to specify as a block:

```suggestion
    data.aws_iam_role.monitor_iam_access_audit,
    data.aws_iam_user.monitor_iam_access_audit,
```